### PR TITLE
Add kie parent and align versions

### PR DESCRIPTION
**Pull request by jiripetrlik**

### CODE REVIEW

File: hacep/pom.xml [hunk 1]
```diff
@@ -5,23 +5,21 @@
   <parent>
     <groupId>org.kie</groupId>
     <artifactId>openshift-kie-parent</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>7.24.0-SNAPSHOT</version>
   </parent>
   <groupId>org.kie</groupId>
   <artifactId>openshift-kie-hacep</artifactId>
-  <version>1.0-SNAPSHOT</version>
+  <version>7.24.0-SNAPSHOT</version>
```
Comment (danielezonca) [inline hacep/pom.xml]: Please remove, only parent version is needed

File: integration/pom.xml [hunk 2]
```diff
@@ -5,104 +5,66 @@
   <parent>
     <groupId>org.kie</groupId>
     <artifactId>openshift-kie-parent</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>7.24.0-SNAPSHOT</version>
   </parent>
   <groupId>org.kie</groupId>
   <artifactId>openshift-kie-integration-test</artifactId>
-  <version>1.0-SNAPSHOT</version>
+  <version>7.24.0-SNAPSHOT</version>
```
Comment (danielezonca) [inline integration/pom.xml]: Please remove, only parent version is needed

File: hacep/pom.xml [hunk 3]
```diff
@@ -69,12 +64,6 @@
 
   <build>
     <finalName>${project.artifactId}</finalName>
```
Comment (danielezonca) [inline hacep/pom.xml]: I think we can remove also this. Wdyt?

File: jdkhttp/pom.xml [hunk 4]
```diff
@@ -5,18 +5,17 @@
   <parent>
     <groupId>org.kie</groupId>
     <artifactId>openshift-kie-parent</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>7.24.0-SNAPSHOT</version>
   </parent>
   <groupId>org.kie</groupId>
   <artifactId>openshift-kie-jdkhttp</artifactId>
-  <version>1.0-SNAPSHOT</version>
+  <version>7.24.0-SNAPSHOT</version>
```
Comment (danielezonca) [inline jdkhttp/pom.xml]: Please remove, only parent version is needed

File: kie-remote/pom.xml [hunk 5]
```diff
@@ -5,9 +5,10 @@
     <parent>
         <artifactId>openshift-kie-parent</artifactId>
         <groupId>org.kie</groupId>
-        <version>1.0-SNAPSHOT</version>
+        <version>7.24.0-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
+    <version>7.24.0-SNAPSHOT</version>
```
Comment (danielezonca) [inline kie-remote/pom.xml]: Please remove, only parent version is needed

File: pom.xml [hunk 6]
```diff
@@ -2,31 +2,30 @@
...
+  <parent>
+    <groupId>org.kie</groupId>
+    <artifactId>kie-parent</artifactId>
+    <version>7.24.0-SNAPSHOT</version> <!-- change with every build -->
+  </parent>
+
   <groupId>org.kie</groupId>
   <artifactId>openshift-kie-parent</artifactId>
-  <version>1.0-SNAPSHOT</version>
+  <version>7.24.0-SNAPSHOT</version>
```
Comment (danielezonca) [inline pom.xml]: Please remove, only parent version is needed

File: sample-hacep-project/sample-hacep-project-kjar/pom.xml [hunk 7]
```diff
@@ -17,13 +18,15 @@
...
-            <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>org.drools</groupId>
             <artifactId>drools-model-compiler</artifactId>
-            <version>${version.org.kie}</version>
         </dependency>
+      <dependency>
+        <groupId>org.drools</groupId>
+        <artifactId>drools-workbench-models-datamodel-api</artifactId>
```
Comment (mariofusco) [inline sample-hacep-project/sample-hacep-project-kjar/pom.xml]: I don't understand why you added this dependency. It works for me even without it.

File: sample-hacep-project/sample-hacep-project-kjar/pom.xml [hunk 8]
```diff
@@ -32,12 +35,14 @@
...
-                <version>${version.org.kie}</version>
                 <extensions>true</extensions>
-            </plugin>
-            <plugin>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <version>${surefire.version}</version>
+              <dependencies>
+                <dependency>
+                  <groupId>org.drools</groupId>
+                  <artifactId>drools-workbench-models-datamodel-api</artifactId>
```
Comment (mariofusco) [inline sample-hacep-project/sample-hacep-project-kjar/pom.xml]: Same thing here. I think this could also be removed.

File: pom.xml [hunk 9]
```diff
@@ -2,31 +2,30 @@
...
     <surefire.forkCount>1</surefire.forkCount>
-    <surefire.version>2.22.0</surefire.version>
-    <version.com.fasterxml.jackson>2.9.9</version.com.fasterxml.jackson>
-    <version.commons.codec>1.12</version.commons.codec>
     <version.javax.cdi>2.0.SP1</version.javax.cdi>
     <version.javax.el>3.0.1-b08</version.javax.el>
     <version.javax.ws.rs>2.1.1</version.javax.ws.rs>
     <version.junit>4.12</version.junit>
-    <version.kafka.client>2.1.1</version.kafka.client>
+    <version.kafka>2.1.1</version.kafka>
```
Comment (danielezonca) [inline pom.xml]: Do we have a property name policy? Maybe `version.org.apache.kafka`?

File: pom.xml [hunk 10]
```diff
@@ -45,6 +44,22 @@
...
+      </dependency>
+      <dependency>
+        <groupId>org.kie</groupId>
+        <artifactId>sample-hacep-project-kjar</artifactId>
+        <version>${version.org.kie}</version>
+      </dependency>
+
       <dependency>
         <groupId>io.thorntail</groupId>
         <artifactId>bom-all</artifactId>
```
Comment (danielezonca) [inline pom.xml]: I know this is not part of this PR but I would like not to import thorntail bom in top level project. Can you please move it to thorntail module? I expect it is needed only there

File: sample-hacep-project/pom.xml [hunk 11]
```diff
@@ -5,9 +5,10 @@
     <parent>
         <artifactId>openshift-kie-parent</artifactId>
         <groupId>org.kie</groupId>
-        <version>1.0-SNAPSHOT</version>
+        <version>7.24.0-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
+    <version>7.24.0-SNAPSHOT</version>
```
Comment (danielezonca) [inline sample-hacep-project/pom.xml]: Please remove, only parent version is needed

File: sample-hacep-project/sample-hacep-project-client/pom.xml [hunk 12]
```diff
@@ -5,22 +5,21 @@
     <parent>
         <artifactId>sample-hacep-project</artifactId>
         <groupId>org.kie</groupId>
-        <version>1.0-SNAPSHOT</version>
+        <version>7.24.0-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
+    <version>7.24.0-SNAPSHOT</version>
```
Comment (danielezonca) [inline sample-hacep-project/sample-hacep-project-client/pom.xml]: Please remove, only parent version is needed

File: sample-hacep-project/sample-hacep-project-kjar/pom.xml [hunk 13]
```diff
@@ -5,9 +5,10 @@
     <parent>
         <artifactId>sample-hacep-project</artifactId>
         <groupId>org.kie</groupId>
-        <version>1.0-SNAPSHOT</version>
+        <version>7.24.0-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
+    <version>7.24.0-SNAPSHOT</version>
```
Comment (danielezonca) [inline sample-hacep-project/sample-hacep-project-kjar/pom.xml]: Please remove, only parent version is needed

**Comment by jiripetrlik** [comment]
@mariofusco 
Thank you for your comments. I've removed unnecessary dependencies.

File: springboot/pom.xml [hunk 14]
```diff
@@ -5,11 +5,11 @@
   <parent>
     <groupId>org.kie</groupId>
     <artifactId>openshift-kie-parent</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>7.24.0-SNAPSHOT</version>
   </parent>
   <groupId>org.kie</groupId>
   <artifactId>openshift-kie-springboot</artifactId>
-  <version>1.0-SNAPSHOT</version>
+  <version>7.24.0-SNAPSHOT</version>
```
Comment (danielezonca) [inline springboot/pom.xml]: Please remove, only parent version is needed

File: thorntail/pom.xml [hunk 15]
```diff
@@ -5,12 +5,12 @@
...
     <groupId>org.kie</groupId>
     <artifactId>openshift-kie-parent</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>7.24.0-SNAPSHOT</version>
   </parent>
 
   <groupId>org.kie</groupId>
   <artifactId>openshift-kie-thorntail</artifactId>
-  <version>1.0-SNAPSHOT</version>
+  <version>7.24.0-SNAPSHOT</version>
```
Comment (danielezonca) [inline thorntail/pom.xml]: Please remove, only parent version is needed

**Comment by jiripetrlik** [comment]
@danielezonca 
Thank you for your comments. It should be fixed now.

**Pull request merged**